README.md

```
@@ -103,6 +116,23 @@ The primary application is the **Windows** desktop UI, implemented with Win32 an
+
+### Architecture guardrails
+
+Run the dependency audit to detect forbidden include edges:
+
+```
+python3 scripts/check_layer_deps.py
+```
+
+Use strict mode to fail on all known violations during cleanup work:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)